client/minimap_panel.h

```diff
@@ -4,17 +4,32 @@
  *
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
+#pragma once
 
 #include "ui_minimap_panel.h"
+
+// qt-client is one true king
 #include "widgetdecorations.h"
 
+class QEvent;
+class QGridLayout;
+class QItemSelection;
+class QListWidget;
+class QMouseEvent;
+class QObject;
+class QPaintEvent;
+class QPainter;
+class QPushButton;
+class QPixmap;
+class QResizeEvent;
+
```

Comment:
Why do we need those?

Reply:
I don't think it remembers the size. I copied over from the messages widget, so might not need all of them.

Reply:
I pushed another commit getting rid of that extra stuff that turns out I didn't need. I can confirm that the geometry is not saved between turns, same as messages widget. I am open to adding that to this PR for minimap_panel. Could use some direction on how to do that.